solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -389,6 +389,12 @@ public ZkController(
     if (overseerEnabled == null) {
       overseerEnabled = EnvUtils.getPropertyAsBool("solr.cloud.overseer.enabled", true);
     }
+    if (overseerEnabled) {
+      log.info("The Overseer is enabled.  It will process all cluster commands & state updates.");
+    } else {
+      log.info(
+          "The Overseer is disabled.  Cluster commands & state updates will happen on any/all nodes.");
+    }
```

Comment:
@psalagnac I updated this PR to log in one spot and to not log in the constructors of the distributed components.